chapter11/spg/Stereotypes.pm

```diff
@@ -0,0 +1,25 @@
+package Stereotypes;
+
+use strict;
+require Exporter;
+
+use vars qw( @ISA @EXPORT_OK );
+
+@ISA = qw( Exporter );
+@EXPORT_OK = qw( dow_to_name moy_to_name );
+
+my @days = qw(ark dip wap sen pop sep kir);
+my @months = qw(diz pod bod rod sip wax lin sen kun fiz nap dep);
+
+sub dow_to_name {
+  my ( $dow ) = @_;
+
+  return $days[$dow] // "INVALID DAY";
```

Comment:
Works, but it might be easily mixed in with valid data returning a string error. It might be better to just return the `undef` for invalid indices, or `die` (e.g. `return $days[$dow] // die 'INVALID DAY'`).